core/src/test/java/com/redhat/lightblue/client/response/TestLightblueResponse.java

```diff
@@ -25,6 +25,17 @@ public void setUp() throws Exception {
         testResponse = new LightblueResponse(initialResponseText);
     }
 
+    @Test(expected = RuntimeException.class)
```

Comment:
I'm assuming it would be this exception, guess you'll let me know if I'm wrong..